drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,60 @@
+:scss
+  .user-permission-report td ul {
+    list-style: none;
+    padding-left: 0;
+  }
+  li {
+    margin-bottom: 0.5em;
+  }
+:ruby
+  title = _('User Permission Report')
+  content_for :title, title
+
+= render 'warehouse_reports/breadcrumbs', tab: 'operational'
+
+.o-page
+  .o-page__header
+    - @prompt = ''
```

Comment:
If this is used for the search placeholder, we might as well set it to something.  Usually instance variables would be set in the controller though `@prompt = 'Search Users'`

Reply:
✅